src/catkin_pkg/package_templates.py

```diff
@@ -309,7 +311,7 @@ def _create_include_macro(package_template):
         if deplist:
             result += '\n# TODO: Check names of system library include directories (%s)' % ', '.join(deplist)
     if includes:
-        result += '\ninclude_directories(\n  %s\n)' % '\n  '.join(includes)
+        result += '\n%s' % '\n  '.join(includes)
```

Comment:
I'll have to brush up on (aka learn) my list parsing (`.append` and `join()`) it seems!